.github/workflows/build.yml

```diff
@@ -0,0 +1,33 @@
+name: Build
+on: [push, pull_request]
+
+jobs:
+  Build:
+    name: "Build on ${{ matrix.name }}"
+    runs-on: ${{ matrix.os }}
```

Comment:
Add `if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository` to only run the workflow once for pull requests